spec/models/wunderground_history_spec.rb

```diff
@@ -54,22 +105,25 @@
       expect(return_object).to be_a WundergroundHistory
     end
 
+    # don't make this one a Wunderground::Error, it's from us
     it "raises errors if data requested prematurely" do
-      wunderground = Wunderground.new(ENV["WUNDERGROUND_KEY"])
-      time = Time.zone.parse('January 22, 2016 12:00:00 -04:00')
+      time = Time.zone.parse('January 23, 2016 at 6pm')
       premature_response = wunderground.history_for(time, 10001)
-      expect {
-        WundergroundHistory.new_from_api(time, premature_response)
-      }.to raise_error(WundergroundHistory::Premature)
+      history = WundergroundHistory.new_from_api(time, premature_response)
+
+      expected_message = 'response is missing desired hour'
+      expect(history.errors).to have(1).error_on(:response)
+      expect(history.errors.messages[:response]).to include(expected_message)
     end
 
-    it "raises errors if rate limited" do
-      wunderground = Wunderground.new(ENV["WUNDERGROUND_KEY"])
-      time = Time.zone.parse('March 1, 2015 00:00:00 -04:00')
+    # make this one a Wunderground::Error
+    it 'raises errors if rate limited' do
+      time = Time.zone.parse('March 1, 2015 at 12am')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.